src/Microsoft.SqlTools.ServiceLayer/ExecutionPlan/Contracts/ExecutionPlanGraph.cs

```diff
@@ -92,6 +96,10 @@ public class ExecutionPlanNode
         /// Top operations table data for the node
         /// </summary>
         public List<TopOperationsDataItem> TopOperationsData { get; set; }
+        /// <summary>
+        /// Rows read metrics for the node
+        /// </summary>
+        public Dictionary<string, string?> RowMetrics { get; set; }
```

Comment:
Is it really rowMetrics or costMetrics?

Reply:
I renamed it to rowMetrics since this dictionary only contains values that pertain to rows for things like:
* Number of Rows Read
* Estimated Number of Rows to be Read
* Actual Number of Rows for All Executions
* Estimated Number of Rows for All Executions

If metrics that didn't just pertain to rows was also included than I would've named it costMetrics in that case.

Reply:
Is there a reason you're passing a dictionary of the row metrics instead of just adding them as additional properties like we have for all the other values? Generally being explicit like that is going to be easier to work with on the TS side